app/controllers/api/v1/packages_controller.rb

```diff
@@ -42,12 +43,13 @@ def show
     api :POST, "/v1/packages", "Create a package"
     param_group :package
     def create
-      @package = Package.new(package_params)
-      @package.offer_id = offer_id
-      if @package.save
-        render json: @package, serializer: serializer, status: 201
-      else
-        render json: @package.errors.to_json, status: 422
+      if package_record
```

Comment:
@swatijadhav if `package_record` is false (i.e. is `existing_package` is false), then we won't render a response?